### ISSUE TITLE
Display blocked sources

### ISSUE THREAD

**Issue by woctezuma** [cowlicks/privacypossum#47]
Right now, it is a bit frustrating in my opinion that blocked sources cannot be checked by the user.

![blocked sources](https://i.imgur.com/MhZK5ft.png)

It would be nice to have something like [uBO-Scope](https://github.com/gorhill/uBO-Scope), so that the user can check that the addon is working as intended. It is also part of the pleasure of using such addons to see what is blocked exactly.

![uBO-Scope](https://i.imgur.com/YwVMbIi.png)

**Comment by rwcorbett** [comment]
Agreed - this would be a nice feature. The blocked sources do not necessarily need to be shown to basic users in the UI, but having them listed in an "advanced view" or as detail on the add-on configuration page would be good for web-devs and power users.

Ghostery has a toggle between advanced/simple view. Privacy Badger (who you had worked for?) lists the trackers as well.

Great work so far!
